views/index.cr

```diff
@@ -1,12 +1,13 @@
 require "ecr/macros"
 
 class Views::Index
-  getter :crystal_repos, :repos_count, :sort
+  getter :crystal_repos, :repos_count, :sort, :filter
 
-  def initialize(repos_data, sort)
+  def initialize(repos_data, sort, filter = nil)
```

Comment:
You are correct, that doesn't make sense. Good point! I will remove that default value.